src/cli.ts

```diff
@@ -142,7 +142,7 @@ async function writeAndImport(
   if (argv.install) {
     await installDeps(parseDeps(contents), {
       prefix: dirname(filepath),
-      registry: argv.registy,
```

Comment:
can touse be passed via env args? To not pollute argv?

Reply:
ZX_REGISTRY / ZX_USERCONFIG ?

Reply:
Done.

Reply:
Awesome 😎